modules/monitoring-certificates.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-certificates.adoc
+
+[id="monitoring-certificates_{context}"]
+= Monitoring certificates
+
+Some client certificate monitoring is done by the Kubernetes API:
+
+----
```

Comment:
We should simplify this! User's don't care how the code looks for rules; the care that we have them and that we tell them what we're looking for. 

A better way to show this might be to say something like: 

> OpenShift monitors certificates for proper validity, for the cluster certificates it issues and manages. Our alerting framework has rules to help us spot when a serious certificate issue is about to occur. These rules consist of the following checks: 
>
> - API Server Client Certificat Expiration < 5 min
> - etc

Reply:
This list does not need to be comprehensive if we then (post this) show how to pull all the certificate rules.